divvy/cloudfilebrowser.scss

```diff
@@ -527,36 +525,37 @@ the License.
         content: '\f15c';
         display: inline;
         font-family: 'FontAwesome';
-        font-size: 13px;
         margin-right: 8px;
     }
 
-    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.filename:before { color: white }
-
-    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.foldername:before { content: '\f07b'; color: burlywood }
+    #services-containers > div.provisioned > div.listTable > div.scrollPanel > ul li.foldername:before { content: '\f07b'; color: #deb887 }
     #services-containers > div.provisioned > div.listTable > ul:first-child {
-        /*position: fixed;*/
-        background: linear-gradient(#555, #333);
-        border-left: 1px solid #777;
-        border-right: 1px solid #333;
-        box-shadow: inset 0 16px 2px rgba(255,255,255,.1), 0 -1px 0 rgba(255,255,255, .3), 0 -2px 0 rgba(0,0,0,.4), 0 1px 1px rgba(0,0,0,0.5);
-        color: #fff;
+        background-color: rgba(255,255,255,1);
```

Comment:
Is it totally necessary to use `rgba()` here or we can we use the hex value?

Reply:
1. the `downloadFile` method will trigger the browser to download the file rather than add the file to the attachments. `fileSelected` is used to add the file to the content item attachments like we want.

2. Not necessary at all. I've changed it to hex.